src/cmd/cmd.cpp

```diff
@@ -193,6 +194,7 @@ void help()
     std::cout << "  -h                     Sync hidden files, do not ignore them" << std::endl;
     std::cout << "  --version, -v          Display version and exit" << std::endl;
     std::cout << "  --logdebug             More verbose logging" << std::endl;
+    std::cout << "  --path                 Path to a folder on a remote server" << std::endl;
```

Comment:
I really like the direction this PR is taking !
I hope this change will make it easier to maintain `nextcloudcmd` in the future